zio-http/src/main/scala/zhttp/http/MediaType.scala

```diff
@@ -0,0 +1,21 @@
+package zhttp.http
+
+case class MediaType(
+  mainType: String,
+  val subType: String,
+  val compressible: Boolean = false,
+  val binary: Boolean = false,
+  val fileExtensions: List[String] = Nil,
```

Comment:
We don't need `val`. It's a `case class`

Reply:
Done.